Sources/TSCUtility/Netrc.swift

```diff
@@ -0,0 +1,176 @@
+import Foundation
+import TSCBasic
+
+
+
```

Comment:
Code that follows imports uses only a single newline in the rest of the codebase.

```suggestion

```